MaterialSkin/Plugin.pm

```diff
@@ -1245,3 +1249,21 @@ sub _userColorHandler {
 }
 
 1;
+
+# Small stub for Plugins::MaterialSkin::Search to use LMS methods if available
+package Plugins::MaterialSkin::Search;
+
+use Slim::Web::Pages::Search;
+
+sub advancedSearch {
+    my ($client, $params) = @_;
+
+    $params->{'searchType'} ||= 'Album';
```

Comment:
Material does not set searchType, as it always wants both tracks and albums. This looks like its defaulting to Albums.

Reply:
You commented out condition in your copy of the implementation. But as LMS is expecting it, we inject it here. Like with the `$client` parameter I'm doing this to keep the interface between the two implementations identical.

Reply:
But will this not restrict the response to albums?

Reply:
...ah, looking at the LMS code it needs this to **also** get albums. My bad.